src/store/modules/search.store.js

```diff
@@ -117,12 +125,13 @@ const actions = {
                     })
                 }
                 break
-            case RESULT_TYPE.LOCATION:
+            case SearchResultTypes.LOCATION:
+            case SearchResultTypes.FEATURE:
                 if (entry.extent.length === 2) {
                     dispatch('zoomToExtent', { extent: entry.extent, dispatcher })
```

Comment:
I would add a maxZoom parameter here, especially for features, as some features (for example : public transports) don't have icons defined on the closest zoom levels.

Reply:
There should already be some safeguard in the selection of the zoom level so that it doesn't go overboard, but I see your point.
I just don't know if we can achieve that without hard-coding a value (which I want to avoid at all cost)

Reply:
there is a maxZoom parameter in the 'zoomToExtent', which zoom to the extent in order to see the whole extent, but stops zooming if it's too close. (For example, in the `@features` special param, it sets the maxZoom to zoom level 8, but it will zoom less if needed).

That being said, it is true it's not pretty having a hard coded value.

if we want to simply make sure there are no issues with the layer shown, we could try to check for the max Zoom supported for the layer (for example `https://sys-wmts.dev.bgdi.ch/1.0.0/ch.bav.haltestellen-oev/default/current/2056/28/9363/6240.png` gives us a 400 error, but tells us the max Zoom level supported is 26. I'm fairly sure the 'maxZoom' supported for a layer should exist somewhere and we could use that)